docs/products/kafka/kafka-connect/howto/jdbc-source-connector-sql-server.rst

```diff
@@ -0,0 +1,121 @@
+
+Furthermore you need to collect the following information about the source SQL Server database upfront:
+
+* ``SQLSERVER_HOST``: The database hostname
+* ``SQLSERVER_PORT``: The database port
+* ``SQLSERVER_USER``: The database user to connect
+* ``SQLSERVER_PASSWORD``: The database password for the ``SQLSERVER_USER``
+* ``SQLSERVER_DATABASE_NAME``: The database name
+* ``SQLSERVER_TABLES``: The list of database tables to be included in Apache Kafka; the list must be in the form of ``table_name1,table_name2``
+
+.. Note::
+
+    If you're using Aiven for SQL Server the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup a SQL Server JDBC source connector with Aiven CLI
+-------------------------------------------------------
+
+The following example demonstrates how to setup an Apache Kafka JDBC source connector to a SQL Server database using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`.
+
+Define a Kafka Connect configuration file
+'''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``jdbc_source_sqlserver.json``) with the following content:
+
+::
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class":"io.aiven.connect.jdbc.JdbcSourceConnector",
+        "connection.url":"jdbc:sqlserver://SQLSERVER_HOST:SQLSERVER_PORT;databaseName=SQLSERVER_DATABASE_NAME;",
+        "connection.user":"SQLSERVER_USER",
+        "connection.password":"SQLSERVER_PASSWORD",
+        "table.whitelist":"SQLSERVER_TABLES",
+        "mode":"JDBC_MODE",
+        "topic.prefix":"KAFKA_TOPIC_PREFIX",
+        "tasks.max":"NR_TASKS",
+        "poll.interval.ms":"POLL_INTERVAL"
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``SQLSERVER_HOST``, ``SQLSERVER_PORT``, ``SQLSERVER_DATABASE_NAME``, ``SQLSERVER_USER``, ``SQLSERVER_PASSWORD`` and ``SQLSERVER_TABLES``: source database parameters collected in the :ref:`prerequisite <connect_jdbc_sqlserver_source_prereq>` phase. 
+* ``mode``: the query mode, more information in the :doc:`dedicated page <../concepts/jdbc-source-modes>`; depending on the selected mode, additional configuration entries might be required.
+* ``topic.prefix``: the prefix that will be used for topic names. The resulting topic name will be the concatenation of the ``topic.prefix`` and the schema and table name.
+* ``tasks.max``: maximum number of tasks to execute in parallel. By default is 1, the connector can use at max 1 task for each source table defined.
+* ``poll.interval.ms``: query frequency, default 5000 milliseconds
+
+Check out the `dedicated documentation <https://github.com/aiven/jdbc-connector-for-apache-kafka/blob/master/docs/source-connector-config-options.rst>`_ for the full list of parameters.
```

Comment:
This seems important. I wonder if you think it makes sense to be as a note so the user can easily see somehow.

Reply:
I believe we have the same text in the three JDBC source example (PG, Mysql and sql-server) I would keep it as it is